NativeAuthSampleApp/ProtectedAPIViewController.swift

```diff
@@ -26,24 +26,29 @@ import MSAL
 import UIKit
 
 class ProtectedAPIViewController: UIViewController {
-    let protectedAPIUrl = "Enter_the_Protected_API_Full_URL_Here"
-    let protectedAPIScopes = ["Enter_the_Protected_API_Scopes_Here"]
+
+    let protectedAPIUrl1: String? = "Enter_the_Protected_API_Full_URL_Here"
+    let protectedAPIScopes1: [String] = ["Enter_the_Protected_API_Scopes_Here"]
+    
+    // Enter the second protected API info Here, if you have one.
+    let protectedAPIUrl2: String? = nil
+    let protectedAPIScopes2: [String] = []
     
     @IBOutlet weak var emailTextField: UITextField!
     @IBOutlet weak var passwordTextField: UITextField!
     @IBOutlet weak var resultTextView: UITextView!
 
```

Comment:
I think the "API1 - perform GET" and "API2 - perform GET" buttons should be enabled only when the user is signed in
<img width="252" alt="Screenshot 2024-05-30 at 15 13 47" src="https://github.com/Azure-Samples/ms-identity-ciam-native-auth-ios-sample/assets/105228698/04499f61-28be-4d21-8cfe-b025be750d93">

Reply:
This was the initial idea, but we decided to change it during the discussion we had over the sample app for multiple access tokens doc.

Reply:
Where this conversation happened? In the multiple access token sample app doc, the screenshots show that the two buttons are disabled when the user is not logged in.

Reply:
On Android these buttons are enabled only when the user signIn. We should do this change on iOS too

Reply:
Fixed